apis/nucleus/src/components/listbox/ListBoxInline.jsx

```diff
@@ -325,6 +320,16 @@ function ListBoxInline({ options, layout }) {
     renderedCallback?.();
   }
 
+  const Title = styled(Typography)(() => {
```

Comment:
Don't move the definition of Title inside ListBoxInline.
It will make react think it always change and need to be re-mounted.
(you can send in the extra info in props instead)
(or maybe re-use the theme.listbox hack but I don't like it and think it should be removed)